client/packages/invoices/src/Returns/modals/OutboundReturn/useDraftOutboundReturnLines.ts

```diff
@@ -45,7 +45,13 @@ export const useDraftOutboundReturnLines = (
         const { id, reasonId, numberOfPacksToReturn, stockLineId, comment } =
           line;
 
-        return { id, stockLineId, reasonId, comment, numberOfPacksToReturn };
+        return {
+          id,
+          stockLineId,
+          reasonId: reasonId ?? '', // TODO
```

Comment:
`reasonId` and `comment` are both optional fields on the `GenerateOutboundReturnLines` response, because they're optional on the InvoiceLine.

However, they're not really optional when it comes to inserting a new OutboundReturnLine... you need to provide a reason to return. So I made the `InsertOutboundReturnLine` input require values here...

Do we usually handle this stuff on the frontend, and pass a definitely-defined value to the backend, or still just accept Option<String> and add new validation that these options aren't None?

Reply:
Would it be simpler for now to not enforce and have as optional ?

Reply:
In front and and back end ?